packages/cypress/install.md

```diff
@@ -0,0 +1,8 @@
+# Cypress rules for Bazel
+
+The Cypress rules run tests under the Cypress e2e testing framework with Bazel.
+
+## Installation
+
+Add the `@bazel/cypress` npm package to your `devDependencies` in `package.json`.
+Then, add `cypress verify` as a `postinstall` script in `package.json`.
```

Comment:
I might need to put something in `/WORKSPACE` too?

Reply:
done